python/tvm/tir/schedule/schedule.py

```diff
@@ -0,0 +1,222 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=unused-import
+"""The TensorIR schedule class"""
+from typing import List, Optional, Union
+
+from tvm._ffi import register_object as _register_object
+from tvm.ir import IRModule, PrimExpr
+from tvm.runtime import Object
+from tvm.tir import Block, For, IntImm, PrimFunc, Var
+
+from . import _ffi_api_schedule
+from .state import ScheduleState, StmtSRef
+
+
+@_register_object("tir.LoopRV")
+class LoopRV(Object):
+    """A random variable that refers to a loop"""
+
+
+@_register_object("tir.BlockRV")
+class BlockRV(Object):
+    """A random variable that refers to a block"""
+
+
+IntRV = PrimExpr  #  A random variable that evaluates to an integer
+
+RAND_VAR_TYPE = Union[IntRV, BlockRV, LoopRV]  # pylint: disable=invalid-name
+
+
+@_register_object("tir.Schedule")
+class Schedule(Object):
+    """The schedule node for TIR"""
```

Comment:
```suggestion
    """The Schedule data structure for TIR"""
```
Is this considered an AST node?

Reply:
oh my bad - it is not an AST node. Your change makes sense to me